libkineto/include/GenericTraceActivity.h

```diff
@@ -26,7 +26,7 @@ constexpr unsigned int kLinkFwdBwd = 1;
 class GenericTraceActivity : public TraceActivity {
 
  public:
-  GenericTraceActivity() = delete;
+  GenericTraceActivity() : activityType(ActivityType::ENUM_COUNT), traceSpan_(NULL) {}
```

Comment:
Do we need this constructor?

Reply:
I am using GenericTraceActivity for the gpu_api and gpu_ops coming out of RoctracerActivityInterface.  I didn't have a source for a TraceSpan.  Also, the cupti api and op activity do not have a TraceSpan, it is NULL.  The only available constructor takes a TraceSpan&.  Setting the type to ENUM_COUNT/invalid seemed proper.

Queue MoreGenericTraceActivity.